docs/apis/avalanchego/apis/p-chain.md

```diff
@@ -1590,6 +1591,7 @@ platform.getStake({
 ```
 
 - `addresses` are the addresses to get information about.
+- `validatorsOnly` 
```

Comment:
please add a description here

Reply:
Funny thing, but I have added a description. Weird that it doesn't show up.

Reply:
Turns out I didn't save it before pushing the commit lol

Reply:
f68303b6c7a98d2ef46dd531688988f5b2d26bae should show up now